railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -231,6 +231,24 @@ def initialize(*args)
           raise Error, "Invalid value for --database option. Supported for preconfiguration are: #{DATABASES.join(", ")}."
         end
 
+        # Fail hard if node or yarn isn't detected on the system
+        if options[:webpack]
+          check_node
+          check_yarn
+        end
+
+        # --webpack uses webpacker, which uses action view helpers
+        # --api skips action view
+        if options[:webpack] && options[:api]
```

Comment:
The only way of using webpacker is with `javascript_pack_tag`? Is it not possible to user webpacker and don't use those helpers?